### ISSUE TITLE
Question: 

### ISSUE THREAD

**Issue by DenchyRS** [jhansche/pybirdbuddy#16]
Hi, I apologies if this is the incorrect way to go about asking a question but I am unsure of how else to.

I am an armature programmer / self taught hobbyist and am trying to use this to make a lil something for my GF however when trying to use your code example in the read me I am receiving this error

```
birds = [c.species.name for c in collections.values()]
AttributeError: 'function' object has no attribute 'values'
```

I was wondering if you could be of any help ? :)

**Comment by jhansche** [comment]
Looks like there's a typo there. We should be invoking `bb.refresh_collections()`. Just add the `()` there.

Fixed in the README [here](c56ccdb2ee7ffb1eb6780ddeca10bc35d9074eab).

**Comment by DenchyRS** [comment]
Thank you very much, appreciate you 🥇
